pymmcore_plus/_client.py

```diff
@@ -39,10 +54,34 @@ def __init__(
 
         self._cb_thread = None
         self._callbacks = set()
-        if callback is not None:
-            self.register_callback(callback)
+        self.register_callback(_CBrelay(self))
 
     def register_callback(self, callback: CallbackProtocol):
+        """Register callback object in proxy process to receive remote callbacks.
+
+        Not to be confused with `mmcore.registerCallback`, which is only used once by
+        CMMCorePlus to receive events coming from the internal C++ CMMcore object.
+
+        Note: RemoteMMCore automatically behaves as a callback receiver, you can
+        connect to any of the signals in the `_CMMCoreSignaler` class:
```

Comment:
This sentence threw me for a loop on the first read. I think this is the first mention of `callback receiver`. 

I was going to suggest a clarification but on further reading I'm struggling to see the use case here. Is there a good reason for this to be public now that you can do `mmcore.<signal>.connect` for all of the possible signals?

Reply:
> Is there a good reason for this to be public now that you can do mmcore.<signal>.connect for all of the possible signals?

well, I can't really think of one! :joy: ... so I'll make it private